source/llassetgen-rendering/main.cpp

```diff
@@ -69,10 +69,10 @@ class Window : public WindowQt {
 
     virtual ~Window() {}
 
-    virtual void initializeGL() override {
+    virtual bool initializeGL() override {
         globjects::init([this](const char * name) {
             return getProcAddress(name);
-        });
+        }, globjects::Texture::BindlessImplementation::Legacy);
```

Comment:
```suggestion
        });
```

Reply:
@scheibel Just FYI: When I remove this line, I get this error:
```
#fatal: error: 0x502, high severity (API)
Error has been generated. GL error GL_INVALID_OPERATION in (null): (ID: 173538523) Generic error
```
on my machine with those specifications:
```
OpenGL Version:  4.3
OpenGL Vendor:   Intel
OpenGL Renderer: Intel(R) HD Graphics 520
```

However, everything renders fine, so I stay with the removed line.